cpp/ql/test/library-tests/ir/ir/raw_ir.ql

```diff
@@ -7,5 +7,7 @@ private import semmle.code.cpp.ir.implementation.raw.PrintIR
 private import PrintConfig
 
 private class PrintConfig extends PrintIRConfiguration {
-  override predicate shouldPrintFunction(Function func) { shouldDumpFunction(func) }
+  override predicate shouldPrintFunction(Declaration decl) {
+    shouldDumpFunction(decl) or decl instanceof GlobalOrNamespaceVariable
```

Comment:
Should we have a `shouldDumpDeclaration` predicate as well? I guess that predicate should exclude global variables with a location that satisfies `locationIsInStandardHeaders` (like we do for `shouldDumpFunction`)